pin/generate_pin

```diff
@@ -22,9 +22,13 @@ OPTIONS:
   -i      set iteration
 
 EXAMPLES:
-  Generate nix pin from nixos-20-03 branch:
+  Generate nix pin from nixos-20.03 branch:
 
-      $(basename "$0") nixos-20-03
+      $(basename "$0") nixos-20.03
```

Comment:
Ah, good catch on this, I don't think we ran into this issue since the pins we used in the repo were already existing in the `pins`. Must be a slight oversight on our end.